frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/align_toolbar_item/custom_text_align_command.dart

```diff
@@ -0,0 +1,105 @@
+import 'package:appflowy/plugins/document/presentation/editor_plugins/plugins.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flutter/material.dart';
+
+final List<CommandShortcutEvent> customTextAlignCommands = [
+  customTextLeftAlignCommand,
+  customTextCenterAlignCommand,
+  customTextRighttAlignCommand,
+];
+
+/// Windows / Linux : ctrl + shift + l
+/// Mac Os          : ctrl + shift + l
+/// Allows the user to align text to the left
+///
+/// - support
+///   - desktop
+///   - web
+///
+final CommandShortcutEvent customTextLeftAlignCommand = CommandShortcutEvent(
+  key: 'Align text to the left',
+  command: 'ctrl+shift+l',
+  macOSCommand: 'ctrl+shift+l',
```

Comment:
I suggest using 'cmd' instead of 'ctrl' on macOS; otherwise, redefining the macOS command again would be meaningless.